packages/program-boilerplate/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "include": [
+    "src/**/*.ts"
+  ],
+  "compilerOptions": {
+    "outDir": "dist",
+    "module": "commonjs",
+    "target": "es2015",
+    "noImplicitAny": false,
```

Comment:
Can you see if this works with `"strict": true`?

See: https://www.typescriptlang.org/docs/handbook/compiler-options.html

strict | boolean | false | Enable all strict type checking options. Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization.
-- | -- | -- | --

Reply:
Updated everything to work with strict :+1: